replica/table.cc

```diff
@@ -2247,7 +2247,7 @@ class compaction_group::table_state : public compaction::table_state {
         return _t.is_auto_compaction_disabled_by_user();
     }
     bool tombstone_gc_enabled() const noexcept override {
-        return _t._tombstone_gc_enabled;
+        return _t.tombstone_gc_enabled()  &&  _cg.tombstone_gc_enabled();
     }
```

Comment:
extra space?

Reply:
That is my old habit of leaving an extra space around binary logical operators, which I will have to get rid of.
I don't think it is a big problem, but I can remove it here, if you insist.

Reply:
It definitely doesn't have to block the merge, can be done later.